Configuration/PyReleaseValidation/python/MatrixInjector.py

```diff
@@ -62,7 +62,7 @@ def __init__(self,opt,mode='init',options=''):
                 self.DbsUrl = "https://"+self.wmagent+"/dbs/int/global/DBSReader"
 
         if not self.dqmgui:
-            self.dqmgui="https://cmsweb.cern.ch/dqm/relval;https://cmsweb-testbed.cern.ch/dqm/relval"
+            self.dqmgui="https://cmsweb.cern.ch/dqm/relval"
```

Comment:
Why was the other URL removed? Looking in CMSSW_7_6_2015-09-27-2300 that URL is still in that release.